cdc/api/util.go

```diff
@@ -97,10 +98,8 @@ func IsHTTPBadRequestError(err error) bool {
 // WriteError write error message to response
 func WriteError(w http.ResponseWriter, statusCode int, err error) {
 	w.WriteHeader(statusCode)
-	_, err = w.Write([]byte(err.Error()))
-	if err != nil {
-		log.Error("write error", zap.Error(err))
-	}
+	tmpl, _ := template.New("").Parse("{{.}}")
+	_ = tmpl.Execute(w, err.Error())
```

Comment:
1. If you want to escape the HTML characters using [`html.EscapeString`](https://pkg.go.dev/html#EscapeString) is sufficient, no need to instantiate a template
2. the CDC API server is not serving HTML, I don't see any reason we need to care about XSS.